.github/workflows/pull-translations.yml

```
@@ -0,0 +1,89 @@
+name: Pull & Commit Translations from Mojito
+
+on:
+  push:
+    paths:
+      # Rebuild on change to workflow itself.
+      - .github/workflows/pull-translations.yml
+    branches:
+      - 320-continuous-localization
```

Accept just before merge:

```suggestion
      - dev
```